service/catalog.go

```diff
@@ -68,14 +122,36 @@ func createCatalog(w http.ResponseWriter, r *http.Request, envId string) (int, e
 		return http.StatusBadRequest, errors.New("Missing field 'url'")
 	}
 
+	if isDuplicateName(catalogModel) {
+		return http.StatusUnprocessableEntity, errors.New("Cannot add catalog with duplicate field `name`")
```

Comment:
Said this in a previous review, but maybe it got lost in the shuffle:
Can you change this error message to "Catalog "foo" already exists"? I think that will be more clear.
obtaining more feddback